controllers/topics/inviteMembers.js

```diff
@@ -57,10 +56,9 @@ module.exports = async (req, res) => {
           return res.status(400).json(response);
         }
 
+        const topic_invitations_id = await uuid();
         const invitations_msg = `${inviter.username} invited you to join ${topics.name} community`;
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `uuid()` function is asynchronous and returns a promise. You should use `await` to ensure that the promise resolves before using its value.
```diff
- const topic_invitations_id = await uuid();
+ const topic_invitations_id = uuid();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->